dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/domain/event/WebViewLogEventSerializerTest.kt

```diff
@@ -85,8 +85,8 @@ internal class WebViewLogEventSerializerTest {
         val jsonObject = JsonParser.parseString(serializedObject).asJsonObject
         assertThat(jsonObject)
             .hasField(KEY_MESSAGE, log.message)
-            .hasField(KEY_SERVICE_NAME, log.service)
-            .hasField(KEY_DATE, log.date)
+            .hasNullableField(KEY_SERVICE_NAME, log.service)
+            .hasNullableField(KEY_DATE, log.date)
```

Comment:
why `date` is nullable if is is required in the schema?

Reply:
oh...is my mistake...should have been the status there